templates/apollo-graphql-react/.platform.template.yaml

```diff
@@ -0,0 +1,31 @@
+
+  notes:
+    - heading: "Features"
+      content: |
+          Node.js 14<br />
+          MariaDB 10.4<br />
+          Websockets<br />
+          Automatic TLS certificates<br />
+
+initialize:
+  repository: git://github.com/platformsh-templates/apollo-graphql-react.git@master
+  config: null
+  files: []
+  profile: apollo
```

Comment:
Since this is a multi-app project, we should include the flags in here to insist on a Medium plan.  See some of the other multi-app templates for what that code is. (I don't recall off hand.  I think xwiki uses it?)

Reply:
Done in a1b5856aef610548c6c6230d36eaba9757296ad7